### PR TITLE
supports multi-arch image manifest

### PR DESCRIPTION
**Pull request by cyclinder**
supports multi-arch image manifest

### CODE REVIEW

File: .github/workflows/image-push-release.yml [hunk 1]
```diff
@@ -35,9 +35,11 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           context: .
+          platforms: linux/amd64
           push: true
           tags: |
-            ${{ steps.docker_meta.outputs.tags }}
+            ${{ steps.docker_meta.outputs.tags }}-amd64
+            ${{ env.IMAGE_NAME }}:${{ github.sha }}
```
Comment (adrianchiris) [inline .github/workflows/image-push-release.yml:42]: this is not needed.

we only create a tag with sha for push to master only

Comment (cyclinder) [reply .github/workflows/image-push-release.yml:42]: Thanks!

**Review by adrianchiris — APPROVED** [review]

**Review by Eoghan1232 — APPROVED** [review]

**Comment by cyclinder** [comment]
Hi @Eoghan1232 , images build failed on amd64. 

https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/actions/runs/3884003915/jobs/6625988380

It seems don't relate to these changes.

**Comment by Eoghan1232** [comment]
> Hi @Eoghan1232 , images build failed on amd64.
> 
> https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/actions/runs/3884003915/jobs/6625988380
> 
> It seems don't relate to these changes?

Hi @cyclinder , yes that the build issue is related to #459 and a PR has been submitted to fix it #460 
this fail is not due to your changes :) 

once the PR is merged, the CICD should then pass

**Pull request merged**